saplings/profile/src/components/NewKey.scss

```diff
@@ -118,10 +122,15 @@
 
           input {
             height: 45px;
-            border: 1px solid #ddd;
+            border: 2px solid var(--color-primary-light);
             border-radius: 5px;
             padding: 10px;
             width: 670px;
+
+            &:focus {
+              outline: none !important;
```

Comment:
Do we need to use `!important` here? It is usually considered to be an antipattern since it makes maintenance difficult and indicates an issue elsewhere, so if it can be solved by other means I would recommend that - otherwise add a comment explaining it. It seems like outline: none should do it, otherwise this could be made a class.

Reply:
Fixed, was able to remove` !import` without affecting the styling